soc/arm/arm/musca_b1/soc.c

```diff
@@ -8,6 +8,43 @@
 #include <init.h>
 #include <soc.h>
 
+/* (Secure System Control) Base Address */
+#define SSE_200_SYSTEM_CTRL_S_BASE	(0x50021000UL)
+#define SSE_200_SYSTEM_CTRL_INITSVTOR1	(SSE_200_SYSTEM_CTRL_S_BASE + 0x114)
+#define SSE_200_SYSTEM_CTRL_CPU_WAIT	(SSE_200_SYSTEM_CTRL_S_BASE + 0x118)
+#define SSE_200_CPU_ID_UNIT_BASE	(0x5001F000UL)
+
+#define NON_SECURE_FLASH_ADDRESS	(0x60000)
```

Comment:
Could these be taken from DTS?

Reply:
Most of them are just an offset of regs or flash address, I am not sure if it can be a kind of device in the tree. Open for more opinions here.

Reply:
@galak ^^